index.js

```diff
@@ -0,0 +1,31 @@
+import DisplaySections from './modules/display_books.modules.js';
+import bookManager from './modules/book_manager.modules.js';
+import initVisitedMenu from './modules/visited_menu.modules.js';
+import timeDisplayer from './modules/dateModule.js';
+
+class App {
+  constructor() {
+    this.form = document.querySelector('form');
+    this.init();
+  }
+
+  init = () => {
+    initVisitedMenu();
+    bookManager.displayBooks();
+    DisplaySections.singlePage();
+    this.setupFormEventListeners();
+    timeDisplayer.updateTime();
+  }
+
+  setupFormEventListeners = () => {
+    this.form.addEventListener('submit', (e) => {
+      e.preventDefault();
+      bookManager.addNewData();
+    });
+  }
+}
+
+const app = new App();
+export default app;
+
+// export {DisplaySections, bookManager, initVisitedMenu}
```

Comment:
- [ ] Kindly remove the commented line of code as it is bad practice to commit commented code. You can visit git history to view old code.